apidoc/Titanium/UI/ProgressBar.yml

```diff
@@ -71,13 +71,13 @@ properties:
   - name: message
     summary: Progress bar text.
     type: String
-
+  - name: tintColor
+    summary: The color shown for the portion of the progress bar that is filled.
```

Comment:
Are you sure? This View also overrides the add-method and here it doesn't redefine properties it doesn't change either:
https://github.com/FokkeZB/titanium_mobile/blob/patch-41/apidoc/Titanium/UI/ProgressBar.yml#L40